src/Tools/ExternalAccess/Razor/Features/Cohost/RazorConfigurationChangedServiceFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageServer;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Cohost;
+
+[ExportCSharpVisualBasicLspServiceFactory(typeof(RazorConfigurationChangedService), WellKnownLspServerKinds.Any), Shared]
```

Comment:
Any should be the default parameter value
```suggestion
[ExportCSharpVisualBasicLspServiceFactory(typeof(RazorConfigurationChangedService)), Shared]
```